docs/api-conventions.md

```diff
@@ -143,8 +175,8 @@ API resources should use the traditional REST pattern:
 
 * GET /&lt;resourceNamePlural&gt; - Retrieve a list of type &lt;resourceName&gt;, e.g. GET /pods returns a list of Pods.
 * POST /&lt;resourceNamePlural&gt; - Create a new resource from the JSON object provided by the client.
-* GET /&lt;resourceNamePlural&gt;/&lt;name&gt; - Retrieves a single resource with the given name, e.g. GET /pods/first returns a Pod named 'first'.
-* DELETE /&lt;resourceNamePlural&gt;/&lt;name&gt;  - Delete the single resource with the given name.
+* GET /&lt;resourceNamePlural&gt;/&lt;name&gt; - Retrieves a single resource with the given name, e.g. GET /pods/first returns a Pod named 'first'. Should be constant time, and the resource should be bounded in size.
+* DELETE /&lt;resourceNamePlural&gt;/&lt;name&gt;  - Delete the single resource with the given name. DeleteOptions may specify gracePeriodSeconds, the optional duration in seconds before the object should be deleted.
```

Comment:
Individual kinds may declare fields which provide a default grace period, and different kinds may have differing kind-wide default grace periods.  A user provided grace period overrides a default grace period, including the zero grace period ("now")